composer.json

```diff
@@ -57,7 +57,7 @@
         "drupal/focal_point": "^2.0",
         "drupal/language_switcher_extended": "^1.1",
         "drupal/link_attributes": "^2.1",
-        "drupal/log_stdout": "dev-3339853-drupal-10-compatibility",
+        "drupal/log_stdout": "dev-8.x-1.x",
```

Comment:
Can you check if D10 compatibility is in the stable version 8.x-1.5?
It seems to be suggested here https://www.drupal.org/node/3339853

Reply:
Updated!